azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -231,6 +233,8 @@ public static void launch(final AzkabanWebServer webServer) throws Exception {
 
     webServer.executorManagerAdapter.start();
 
+    webServer.executionLogsCleaner.start();
```

Comment:
Since this is plugged specifically into `AzkabanWebServer`, could you please verify that it will also work properly in `AzkabanSingleServer`? I think it should, since solo server effectively includes web server, but can we double-check just to make sure? Thanks!